packages/dashboard/src/vaadin-dashboard-section.js

```diff
@@ -203,6 +210,32 @@ class DashboardSection extends DashboardItemMixin(ElementMixin(ThemableMixin(Pol
       this.setAttribute('role', 'section');
     }
   }
+
+  connectedCallback() {
```

Comment:
Would it make sense to add this to `DashboardItemMixin` so that we can shared the logic between sections and widgets and mixins?

Reply:
Removed the event mechanism completely as per [Tomi's suggestion](https://github.com/vaadin/web-components/pull/8885#discussion_r2026412885). It now mimics how the i18n is synchronized.